db/carrier_icons.yml

```diff
@@ -2,11 +2,52 @@
   name: ups
   label: UPS
   group: other
+-
+  name: usps
+  label: USPS
+  group: other
+-
+  name: sendle
+  label: Sendle
+  group: other
+  
+-
+  name: poste-italiane
+  label: PosteItaliane
+  group: other
+-
+  name: mondial-relay
+  label: MondialRelay
+  group: other
 -
   name: fedex
-  label: FedEx
+  label: Fedex
+  group: other
+-
+  name: evri
+  label: Evri
+  group: other
+-
+  name: dpd
+  label: Dpd
   group: other
 -
   name: dhl
   label: DHL
   group: other
+-
+  name: correos
+  label: Correos
+  group: other
+-
+  name: colissimo
+  label: Colissimo
+  group: other
+-
+  name: chronopost
+  label: Chronopost
+  group: other
+-
+  name: canadapost
```

Comment:
Would you be able to add a canadapost_symbol.svg variant?

https://www.canadapost-postescanada.ca/cpc/en/our-company/news-and-media/media-centre/canada-post-logos.page

![CleanShot 2024-02-02 at 12 47 00@2x](https://github.com/Checkout-Blocks/shipping_carrier_icons/assets/3484527/92d7af2d-511d-4e8c-b370-0111118d35fc)